app/src/main/java/com/example/taskyy/ui/screens/ReminderScreen.kt

```diff
@@ -0,0 +1,332 @@
+            }
+        ) {
+            Text(
+                text = dateString,
+                color = Color.Black,
+                fontSize = 15.sp
+            )
+
+            ChevronForward(topPadding = 0)
+        }
+    }
+}
+
+@Composable
+fun ItemDescription() {
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
+        modifier = Modifier.clickable
+        {
+            //navigate to item description page
+        }
+    ) {
+        Text(
+            text = "Reminder Description",
+            modifier = Modifier.padding(start = 20.dp, top = 20.dp),
+            fontSize = 15.sp
+        )
+        ChevronForward(20)
+    }
+}
+
+@Composable
+fun RowDivider(topPadding: Int) {
+    Row(
+        modifier = Modifier.padding(top = topPadding.dp)
+    ) {
+        Box(
+            modifier = Modifier
+                .weight(1f)
+                .height(1.dp) // Adjust the height of the divider as needed
+                .padding(
+                    bottom = 4.dp,
+                    start = 20.dp,
+                    end = 20.dp
+                ) // Adjust the padding to control the distance between the divider and the content
+        ) {
+            HorizontalDivider(color = Color.LightGray) // Divider with black color
+        }
+    }
+}
+
+@Composable
+fun NewItemRow() {
+    Row(
+        modifier =
+        Modifier
+            .padding(top = 100.dp, start = 20.dp)
+            .clickable {
+                //navigate to name screen
+            },
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        WhiteCircleWithBlackBorder()
+        Text(
+            text = "New Reminder",
+            fontSize = 25.sp,
+            fontWeight = FontWeight.Bold,
+            modifier = Modifier.padding(start = 10.dp)
+        )
+        ChevronForward(0)
+    }
+}
+
+@Composable
+fun ChevronForward(topPadding: Int) {
+    Column(
+        horizontalAlignment = Alignment.End,
+        modifier = Modifier
+            .padding(top = topPadding.dp)
+            .fillMaxWidth()
+    ) {
+        Image(
+            painter = painterResource(id = R.drawable.chevron_forward),
+            contentDescription = "proceed forward arrow",
+            modifier = Modifier.padding(end = 15.dp)
+        )
+    }
+}
+
+@Composable
+fun AlarmIconWithBackground() {
+    Box(
+        modifier = Modifier
+            .size(32.dp) // Adjust the size of the vector asset as needed
+            .background(
+                color = Color(android.graphics.Color.parseColor("#f2f6fc")),
+                shape = AbsoluteRoundedCornerShape(5.dp, 5.dp, 5.dp, 5.dp)
+            ),
+        contentAlignment = Alignment.Center
+
+    ) {
+        Image(
+            painter = painterResource(
+                id = R.drawable.alarm_icon
+            ),
+            contentDescription = "Alarm icon",
+            colorFilter = ColorFilter.tint(color = Color(android.graphics.Color.parseColor("#aeb4bf")))
+        )
```

Comment:
Also for icons, always use `Icon` instead of `Image`. That way, you can easily choose a tint (without any color filter magic):
```
Icon(
    imageVector = ImageVector.vectorResource(R.drawable.alarm_icon),
    tint = Color(0xFFAEB4BF)
)
```